kodilla-hibernate/src/test/java/com/kodilla/hibernate/manytomany/dao/CompanyDaoTestSuite.java

```diff
@@ -52,14 +56,68 @@ public void testSaveManyToMany(){
         Assert.assertNotEquals(0,greyMatterId);
 
         //CleanUp
-        /*
+
         try {
             companyDao.delete(softwareMachineId);
             companyDao.delete(dataMaestersId);
             companyDao.delete(greyMatterId);
         } catch (Exception e) {
             //do nothing
         }
-        */
+
+    }
+
+    @Test
+    public void testEmployeeNamedQuery(){
+        //Given
+        Employee johnSmith = new Employee("John", "Smith");
+        Employee stephanieClarckson = new Employee("Stephanie", "Clarckson");
+        Employee lindaKovalsky = new Employee("Linda", "Kovalsky");
+
+        employeeDao.save(johnSmith);
+        int employeeOneId = johnSmith.getId();
+        employeeDao.save(stephanieClarckson);
+        int employeeTwoId = stephanieClarckson.getId();
+        employeeDao.save(lindaKovalsky);
+        int employeeThreeId = lindaKovalsky.getId();
+
+        //when
+        List<Employee> showLastname = employeeDao.retrieveEmployeeLastname("Kovalsky");
+
+        //Then
+        Assert.assertEquals("Kovalsky", lindaKovalsky.getLastname());
+        Assert.assertEquals(1, showLastname.size());
+
+        //CleanUp
+        employeeDao.delete(employeeOneId);
+        employeeDao.delete(employeeTwoId);
+        employeeDao.delete(employeeThreeId);
+    }
+
+    @Test
+    public void testCompanyNamedNativeQuery(){
+        //Given
+        Company softwareMachine = new Company("Software Machine");
+        Company dataMaesters = new Company("Data Maesters");
+        Company greyMatter = new Company("Grey Matter");
+
+        companyDao.save(softwareMachine);
+        int softwareMachineId = softwareMachine.getId();
+        companyDao.save(dataMaesters);
+        int dataMaestersId = dataMaesters.getId();
+        companyDao.save(greyMatter);
+        int greyMatterId = greyMatter.getId();
+
+        //When
+        List<Company> showCompanyName = companyDao.retrieveCompanyNameFirstLetters("Gre%");
```

Comment:
tutaj jest problem z Gre% - ten % powienien byc dodawany w query w NamedNativeQuery w Company.
LIKE - to jest ok, ale uzyj z komenda CONCAT('%',:COMPANY_NAME)